src/main/resources/templates/index.html

```diff
@@ -17,11 +17,12 @@
             <h1 class="display-4">Welcome to BulkBuy</h1>
         </div>
     </div>
-</main>
 
-<div class="container">
+
+<div class="container outerContentWrapper">
```

Comment:
Added the outerContentWrapper CSS class to constrain the content and keep it from overflowing into other elements, such as the footer.